package.json

```diff
@@ -12,12 +12,13 @@
     "bower": "1.8.8",
     "color-convert": "2.0.1",
     "colors": "1.4.0",
+    "coffeescript": "^2.5.1",
```

Comment:
this isn’t a new dependency. It was pulled in transitively before. Now it’s no longer pulled in transitively but we still need it.